cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -65,12 +65,14 @@ mode and use a `cloud-init` disk, the virtual machine will lose its network
 connectivity after being restarted. As a workaround, remove the `HWADDR` line
 in the file `/etc/sysconfig/network-scripts/ifcfg-eth0`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1708680[*BZ#1708680*])
-+
-[NOTE]
-====
-Avoid this issue by using the recommended `masquerade` binding method for
-the default Pod network.
-====
+
+* Masquerade does not currently work with CNV. Due to an upstream issue,
```

Comment:
Just curious if there should there be a BZ link at the end of this, like how there is for all the other known issues.

Reply:
No BZ link as it is an upstream issue. If you know if github links are permitted I can add one of those.

Reply:
Actually I can add a BZ link for this blurb. I was confusing it with the other missing link below.